src/toil/test/cwl/cwlTest.py

```diff
@@ -660,55 +750,185 @@ def test_streamable(self):
             self.assertEqual(f.read().strip(), "When is s4 coming out?")
 
 
+@needs_cwl
```

Comment:
The unit tests that I added start here.  

I ran black on the cwlTest.py file and I guess it wasn't formatted in a while.  

I can roll this back if you all want.